packages/client/src/views/SysAdmin/Performance/OperationalReport.tsx

```diff
@@ -169,6 +169,18 @@ const RowLink = styled(LinkButton)`
   padding-right: 15px !important;
 `
 
+const status: Array<string> = [
+  'IN_PROGRESS',
+  'DECLARED',
+  'REJECTED',
+  'VALIDATED',
+  'WAITING_VALIDATION',
+  'REGISTERED'
+]
+if (!window.config.EXTERNAL_VALIDATION_WORKQUEUE) {
```

Comment:
Instead of removing the `WAITING_VALIDATION` option from here, let's filter it out in `StatusWiseApplicationCountView` when the flag is off.